src/rebar_pkg_resource.erl

```diff
@@ -95,7 +95,7 @@ extract(TmpDir, CachePath) ->
     {"metadata.config", Meta} = lists:keyfind("metadata.config", 1, Files),
     {Files, Contents, Version, Meta}.
 
-checksums(Pkg={pkg, _Name, _Vsn, Hash}, Files, Contents, Version, Meta, State) ->
+checksums(Pkg={{pkg, _Name, _Vsn, Hash}, _Registry}, Files, Contents, Version, Meta, State) ->
```

Comment:
Any internal format change (adding or removing an element to the tuple) broke things. In this case it appears to wrap the package into another tuple.